src/reducers/__test__/articles.test.js

```diff
@@ -0,0 +1,459 @@
+/* global describe, it */
+
+/*
+  Includes
+    1. TESTING of `relatedArticles` reducer 
+    2. TESTING of `articlesByUuids` reducer
+    3. TESTING of `featureArticles` reducer
+*/
+
+'use strict'
+
+import { expect } from 'chai'
+import * as reducers from '../articles'
+import * as types from '../../constants/action-types'
+
+/* -------- MOCK DATA -------- */
+const initialState = {}
+
+const mockActions = {
+  notReconized: {
+    type: 'This action is not be recognized by the reducer'
+  },
+  [types.FETCH_RELATED_ARTICLES_REQUEST]: {
+    type: types.FETCH_RELATED_ARTICLES_REQUEST,
+    id: 'act_article_id_01',
+    relatedIds: [ 'act_related_id_01' ],
+    url: 'act_api_url_01'
+  },
+  [types.FETCH_RELATED_ARTICLES_FAILURE]: {
+    type: types.FETCH_RELATED_ARTICLES_FAILURE,
+    id: 'act_article_id_01',
+    relatedIds: [ 'act_related_article_id_01' ],
```

Comment:
The reducer will not save `relatedIds` into state, but it's included in input action

`actions/articles.js`:

```
function failToReceiveRelatedArticles(id, relatedIds, error) {
  return {
    type: types.FETCH_RELATED_ARTICLES_FAILURE,
    id,
    relatedIds,
    error,
    failedAt: Date.now()
  }
}
```